AGENTS.md

```
@@ -1,169 +1,234 @@
 
-## Build
+---
 
-```bash
-# build the UI
-yarn workspace @kaoto/kaoto build
+## Repository Structure
 
-# build the public component library outputs
-yarn workspace @kaoto/kaoto build:lib
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)